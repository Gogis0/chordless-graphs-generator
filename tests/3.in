1
5
5 6
0 1
1 2
2 3
3 4
4 0
1 4

