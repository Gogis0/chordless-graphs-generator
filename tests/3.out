G contains a critical edge
